homework/03LessonThree.js

```diff
@@ -57,19 +60,19 @@ var ternaryResult4 = ""     ? 'first' : 'second';
 
 // Problem 4:
 // What is the value of ternaryResult1
-var problemFourAnswer;
+var problemFourAnswer = true ? 'first' : 'second';
```

Comment:
Let's talk about this ^ it passes the test, but it's not how we wanted you to answer. We wanted you to 'choose' an answer like this

```javascript
var problemFourAnswer = 'some answer';
```